python/eos/tasks.py

```diff
@@ -15,6 +15,7 @@
 # this program; if not, write to the Free Software Foundation, Inc., 59 Temple
 # Place, Suite 330, Boston, MA  02111-1307  USA
 
+from pypmc import density
```

Comment:
Arf, this automatic behavior is really boring. This has nothing to do there. VSCode tries to be too smart...